docs/api/migration-guides/qiskit-runtime-options.mdx

```diff
@@ -0,0 +1,46 @@
+
+# Migrate options
+
+All `backend.run` options are now available through the Qiskit Runtime primitives, but there are additional options available with the V2 primitives.  See the [API reference](/api/qiskit-ibm-runtime/options) for the full list of available options. See [End-to-end examples](qiskit-runtime-examples) and [Advanced Qiskit Runtime options](../../run/advanced-runtime-options) for more information about specifying V2 primitives options. 
+
+| backend.run options | V2 Primitive options                                                                                                  | Notes                                                                                                                                |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| job_name            | N/A: Use job_tags instead.                                                                                            | ibmq-provider only                                                                                                                   |
+| job_share_level     | N/A: This is no longer supported.                                                            | ibmq-provider only                                                                                                                   |
+| job_tags            | options.environment.job_tags                                                                                          |                                                                                                                                      |
+| experiment_id       | N/A: Use job_tags instead.                                                                                            | ibmq-provider only                                                                                                                   |
+| header              | You can attach metadata to circuits. The same metadata is returned in the result metadata, under “circuit_metadata”.  |                                                                                                                                      |
+| shots               | Any of the following: <br></br>• run([(…, shots)]) <br></br>• run(…, shots=) <br></br>• options.default_shots                                                |                                                                                                                                      |
+| memory              | Use result.data.classical_register_name.get_counts() to get counts. <br></br>Use result.data.classical_register_name.get_bitstrings() to get per-shot measurements.    |                                                                                          |
```

Comment:
```suggestion
| memory              | Use result.data.<classical_register_name>.get_counts() to get counts. <br></br>Use result.data.<classical_register_name>.get_bitstrings() to get per-shot measurements.    |                                                                                          |
```

Reply:
I had it like that, but putting <> in the table makes it fail the preview test, so I removed them.  I'm not sure how to solve the problem.  I'll try some things.

Reply:
backslash (`\`) is the common escape character, so maybe that'll work